### ISSUE TITLE
New Assembler not identifying undefined symbols

### ISSUE THREAD

**Issue by RogerDoering** [mipsym/MIPSym-issues#3]
**Describe the bug**
new assembler not flagging undefined symbols.

**To Reproduce**
assemble any code with an undefined label.

**Assembly Source Code that Causes the Problem**
```
    .data
your_program_here:
    .code
    la    $a0,fred   #fred is undefined. 
```

**Expected behavior**
I expect an assembly error.

**Platform(s) affected (type an 'X' between the square brackets):**
 - [x] Windows
 - [ ] Mac
 - [ ] Linux (please indicate which distribution and version)

**Program(s) affected (type an 'X' between the square brackets):**
 - [ ] MIPSym
 - [ ] MIPSymCLI
 - [x] MIPSym Assembler

**MIPSym Version Affected:**
 2.05.1519

**Additional context**
This doesn't happen on the builder version.

**Comment by ddalcino** [comment]
This is now fixed in the main branch; the fix will go out in the next release.

**Comment by ddalcino** [comment]
Fixed in 2.05.1520
